CHANGELOG.md

```diff
@@ -3,6 +3,70 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.9.0
+
+_Jul 5, 2024_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 📊 Add Heatmap charts
```

Comment:
We didn't release the package, so it might be better not to announce it?

Reply:
ok, then in that case I'll remove it from the announcements section.